src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = days * 40;
+
+  if (days >= 7) {
+    return cost - 50;
```

Comment:
Use a constant to represent the long-term rental threshold and the associated discount, instead of hardcoding the values '7' and '50'.

Reply:
Still not fixed